.github/workflows/ci.yaml

```diff
@@ -8,8 +8,9 @@ jobs:
   test-linux:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        node: [ 10, 12, 14, 15, 16]
+        node: [ 12, 14, 16, 17, 18 ]
```

Comment:
Consider waiting to add Node 18 until tests for Node 18 pass?

Reply:
I was thinking about that too. I think I'll keep it this way because it mirrors the `system-test.sh` side. The check isn't required to pass now. I'll make it a required check as soon as tests for node 18 passes.

FWIW, I'm planning to temporarily use the `0.8.0-beta.0` version of `source-map` while waiting for the upstream issue to be fixed.

Reply:
SGTM